sql/init.sql

```diff
@@ -0,0 +1,20 @@
+-- create schema used for API
+create schema api;
+
+-- create table used for log events
+create table api.events (
+	id serial primary key,
+	event jsonb not null
+);
+
+-- create web user w/ read only auth
+create role web_anon nologin;
```

Comment:
as we plan to discontinue sending a customized, pre-baked JWT with the claim `"role": "event_logger"`, I would expect we need to bless the web_anon role with grants like those given to `event_logger` below?

Reply:
>we need to bless the web_anon role with grants like those given to `event_logger`

Thanks for the reminder! I hadn't tested writing values into logserver (only viewing existing data). I'll make sure to test that the `web_anon` role can write events without authentication